make/proto.mk

```diff
@@ -24,12 +24,19 @@ proto.gen.doc:
 
 .PHONY: proto.gen.init
 proto.gen.init:
-	 go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
-	 go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
 
 .PHONY: proto.gen.code
 proto.gen.code:
 	$(DOCKER) build -t layotto/protoc $(ROOT_DIR)/docker/proto && \
 	$(DOCKER) run --rm \
 		-v  $(ROOT_DIR)/spec/proto/runtime/v1:/api/proto \
 		layotto/protoc
+
+.PHONY: proto.comments
+proto.comments:
+	curl -fsSL \
+		"https://github.com/bufbuild/buf/releases/download/v1.6.0/buf-$$(uname -s)-$$(uname -m)" \
+		-o "$(OUTPUT_DIR)/buf"
+	buf lint $(ROOT_DIR)
```

Comment:
need to add a new line at the end of yaml file.

Reply:
Do we need to  download the file every time when user executes the `make proto.comments`?

Reply:
suggest changing like this:
```shell
.PHONY: proto.comments
proto.comments:
ifeq (,$(shell which buf))
	@echo "===========> Installing buf linter"
	@curl -fsSL \
		"https://github.com/bufbuild/buf/releases/download/v1.6.0/buf-$$(uname -s)-$$(uname -m)" \
		-o "$(OUTPUT_DIR)/buf"
	@sudo install -m 0755 $(OUTPUT_DIR)/buf /usr/local/bin/buf
endif
	@echo "===========> Running buf linter"
	buf lint $(ROOT_DIR)
```

Reply:
> suggest changing like this:
> 
> ```shell
> .PHONY: proto.comments
> proto.comments:
> ifeq (,$(shell which buf))
> 	@echo "===========> Installing buf linter"
> 	@curl -fsSL \
> 		"https://github.com/bufbuild/buf/releases/download/v1.6.0/buf-$$(uname -s)-$$(uname -m)" \
> 		-o "$(OUTPUT_DIR)/buf"
> 	@sudo install -m 0755 $(OUTPUT_DIR)/buf /usr/local/bin/buf
> endif
> 	@echo "===========> Running buf linter"
> 	buf lint $(ROOT_DIR)
> ```

Done.